pkgs/build-support/setup-hooks/wrap-apps-hook/wrap-apps-hook.sh

```diff
@@ -0,0 +1,79 @@
+# shellcheck shell=bash
+appsWrapperArgs=()
```

Comment:
Do we want to use this variable, or something else? Originally I had `makeWrapperArgs`, but that will conflict with the current (rare) usage of `makeWrapperArgs` of Python packages.